docs/rules/valid-title.md

```diff
@@ -168,9 +168,34 @@ Examples of **correct** code for this rule with the `{ "mustMatch": { "describe"
 ```js
 // The describe title is checked with the default regex, so it's valid
 describe('foo', () => {
-  // This check succeeds because the title matches the regex  
+  // This check succeeds because the title matches the regex
   it('should be a number.', () => {
     expect(1).toBeNumber()
   })
 })
 ```
+
+Note: If you'd like to use a function or class names inside `describe`, `test` or `it` blocks as a parameter, you must enable vitest's type checking.
+
+To enable typechecking for vistest make sure settings key is added in your configuration
```

Comment:
Typo?

Reply:
@so1ve, how do you want it to be phrased?

Reply:
I mean `vistest`

Reply:
ooh good catch thanks!